tests/e2e/import-create-account.test.js

```diff
@@ -124,7 +124,7 @@ describe('Import/Create', () => {
 
       const value = await page.evaluate((el) => el.textContent, plugBanner);
 
-      expect(value).toMatch('Alpha - 0.5.3');
+      expect(value).toMatch('Alpha - 0.6.0');
```

Comment:
Is there a way to avoid having this hardcoded? Like instead of toMatch have it greater than?

Reply:
@alinasytnik I noticed that under Sonic testnet there are limitations, like user cannot access to certain tab and cannot search for tokens, and only can add Custom Token. Would you like at some point add testing for those scenarios also?
I would also include a send transaction just to confirm everything allowed under this network is working ok. - related to this comment

@karlamiesesfleek 

Yeah, I agree with you. I'm checking only the NFT tab on disabling and definitely should check it for all disabled functions. 
Regarding adding and sending tokens on sonic network, I'm currently working on it and will let you know when i finish it :)